sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java

```diff
@@ -72,24 +55,52 @@ public void createSmsClienttWithConnectionString() {
             .buildClient();
     }
 
-    /**
-     * Sample code for creating a sync Communication Identity Client using AAD authentication.
-     *
-     * @return the Communication Identity Client.
-     */
-    public SmsClient createCommunicationIdentityClientWithAAD() {
-        // You can find your endpoint and access key from your resource in the Azure Portal
-        String endpoint = "https://<RESOURCE_NAME>.communication.azure.com";
+    public void sendMessageToOneRecipient(SmsClient smsClient) {
+        //Send an sms to only one phone number
+        String to = "<to-phone-number>";
 
-        // Create an HttpClient builder of your choice and customize it
-        HttpClient httpClient = new NettyAsyncHttpClientBuilder().build();
+        // to enable a delivery report to the Azure Event Grid
+        SmsSendOptions options = new SmsSendOptions();
+        options.setDeliveryReportEnabled(true);
+        //addionaly you can ad a tag you wish to identify the messages for this tag.
+        options.setTag("Tag"); /* Optional */
+
+        // Send the message to a list of  phone Numbers and check the response for a messages ids
+        SmsSendResult response = smsClient.send(
+            "<leased-phone-number>",
+            to,
+            "your message",
+            options /* Optional */);
 
-        SmsClient smsClient = new SmsClientBuilder()
-            .endpoint(endpoint)
-            .credential(new DefaultAzureCredentialBuilder().build())
-            .httpClient(httpClient)
-            .buildClient();
+        System.out.println("MessageId: " + response.getMessageId());
+
+    }
+
+    public void sendMessageToMultipleRecipients(SmsClient smsClient) {
+        //Send an sms to multiple phone numbers
```

Comment:
Remove all these comments. We should keep these to minimum

Reply:
Wont be easier for the developer to have in here some coomment about tag and the enable delivery report? That way they will only need to look at the samples and answer themselfs  what are those methods.